requirements-test.txt

```diff
@@ -680,4 +680,4 @@ xxhash==3.5.0
 yarl==1.17.1
     # via aiohttp
 zstandard==0.23.0
-    # via lm-eval
```

Comment:
not sure what is going on here, or what the difference is

Reply:
I ran `autopep8 --in-place --max-line-length 120 requirements-test.txt` to see what would happen